<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>encoderForm</class>
 <widget class="QWidget" name="encoderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>encoder</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>241</height>
    </size>
   </property>
   <row>
    <property name="text">
     <string>Alpha AxisE</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Alpha WormE</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Alpha Motor</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>RemainingE</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Delta AxisE</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Delta WormE</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Delta Motor</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>RemainingE</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>lecture</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>position</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
